### ISSUE TITLE
Specify English as document language in html

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-web#78]
I ran YACS through the following accessibility checker: http://wave.webaim.org/report#/yacs.cs.rpi.edu

The first apparent issue was the lack of document language. Specifying the document language as English will not only make the page more accessible to people who use screen readers, but also to people whose first language is not English and may rely on automatic translation.

This can be done by adding a `lang` attribute to the `html` tag.
